README.md

```diff
@@ -2,6 +2,10 @@
 
 [![License](https://img.shields.io/github/license/qiskit-community/qiskit-algorithms.svg?style=popout-square)](https://opensource.org/licenses/Apache-2.0)<!--- long-description-skip-begin -->[![Build Status](https://github.com/qiskit-community/qiskit-algorithms/workflows/Algorithms%20Unit%20Tests/badge.svg?branch=main)](https://github.com/qiskit-community/qiskit-algorithms/actions?query=workflow%3A"Algorithms%20Unit%20Tests"+branch%3Amain+event%3Apush)[![](https://img.shields.io/github/release/qiskit-community/qiskit-algorithms.svg?style=popout-square)](https://github.com/qiskit-community/qiskit-algorithms/releases)[![](https://img.shields.io/pypi/dm/qiskit-algorithms.svg?style=popout-square)](https://pypi.org/project/qiskit-algorithms/)[![Coverage Status](https://coveralls.io/repos/github/qiskit-community/qiskit-algorithms/badge.svg?branch=main)](https://coveralls.io/github/qiskit-community/qiskit-algorithms?branch=main)<!--- long-description-skip-end -->
 
+> [!WARNING]  
+> **Qiskit Algorithms is not longer officially supported by IBM Quantum**.  
```

Comment:
```suggestion
> **Qiskit Algorithms is no longer officially supported by IBM Quantum**.  
```

And should we just drop the Quantum at the end and more simply state `Qiskit Algorithms is no longer officially supported by IBM`